linera-service/src/cli_wrappers/util.rs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use pathdiff::diff_paths;
+use tokio::process::Command;
+
+use crate::util::CommandExt;
+
+pub async fn get_github_root() -> Result<String, anyhow::Error> {
+    let github_root = Command::new("git")
```

Comment:
Let's discuss how we can avoid calling `git` and making assumption on the current directory for non-test use cases.

Reply:
Will change to use the `git2` crate as it is a more elegant solution in my opinion, but open to other ideas

Reply:
It's not a question of wrapper library vs command line.

Reply:
I know, but `git2` is not supposed to be a wrapper library, AFAIK. From what I researched, it is "a set of Rust bindings to the libgit2 library, which is a portable, pure C implementation of Git core methods provided as a re-entrant linkable library."
But I understand that the discussion is how we can avoid calling it at all :)